src/app/conversion.ts

```diff
@@ -38,43 +40,42 @@ function targetUnits(quantity) {
   }
 }
 
-function renderMagnitude(units, magnitude, fractions = true) {
-  if (!magnitude) return magnitude;
+function renderMagnitude(units: string, magnitude: number, fractions = true) : string {
+  if (!magnitude) return null;
   if (magnitude >= 50) {
     magnitude = magnitude / 5;
     magnitude = Math.round(magnitude) * 5;
     magnitude = Number(magnitude.toPrecision(3));
     magnitude = Math.round(magnitude);
-    return magnitude.toFixed();
+    return magnitude.toString();
```

Comment:
TODO: Switch back to using `magnitude.toFixed()` here.  We don't have any confirmed sightings of differences due to this change, and there isn't a clear reason it was introduced.